trino/client.py

```diff
@@ -686,6 +686,10 @@ def execute(self, additional_http_headers=None) -> TrinoResult:
         if self.cancelled:
             raise exceptions.TrinoUserError("Query has been cancelled", self.query_id)
 
+        if self._experimental_python_types:
+            new_header = {constants.HEADER_CLIENT_CAPABILITIES: 'PARAMETRIC_DATETIME'}
```

Comment:
nit: extract constant for header value. Do we do this for other well-named header values as well?